process/project-lifecycle-documents/osv-schema_graduation_stage.md

```diff
@@ -0,0 +1,92 @@
+
+OSV Schema has had 62 contributors from 18 different organizations.
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be code needed to deliver OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  * The mission of OSV is to develop a standard interchange format for describing vulnerabilities in open source packages.
+  * The OSV schema provides a human and machine readable data format to describe vulnerabilities in a way that precisely maps to open source package versions or commit hashes.
+
+### Project adoption
+The project must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+
+The OSV Schema is currently exported by:
+- [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+- [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
+- [Chainguard](https://packages.cgr.dev/chainguard/osv/all.json)
+- [Curl](https://curl.se/docs/vuln.json)
+- [GitHub Security Advisories](https://github.com/github/advisory-database)
+- [Global Security Database](https://github.com/cloudsecurityalliance/gsd-database)
+- [Go Vulnerability Database](https://github.com/golang/vulndb)
+- [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+- [LoopBack Advisory Database](https://github.com/loopbackio/security/tree/main/advisories)
+- [Malicious Packages Repository](https://github.com/ossf/malicious-packages)
+- [Mageia Advisories](https://advisories.mageia.org/)
+- [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
+- [OSV.dev maintained converters](https://google.github.io/osv.dev/data/#converted-data) (Debian, Alpine, NVD)
+- [PyPI Advisory Database](https://github.com/pypa/advisory-database)
+- [Python Software Foundation Database](https://github.com/psf/advisory-database)
+- [RConsortium Advisory Database](https://github.com/RConsortium/r-advisory-database)
+- [Red Hat](https://security.access.redhat.com/data)
+- [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+- [Rust Advisory Database](https://github.com/RustSec/advisory-db)
+- [SUSE](https://www.suse.com/support/security/)
+- [Ubuntu](https://github.com/canonical/ubuntu-security-notices/)
+
+### Release cadence
+The project must be able to show a consistent release cadence.
+  * https://github.com/ossf/osv-schema/releases
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * https://github.com/ossf/osv-schema/blob/main/CHARTER.md
+
+Have a defined and documented roadmap and annual goals for the project
+  * "link to roadmap and goals"
```

Comment:
Note to self: link to a project at https://github.com/ossf/osv-schema/projects?query=is%3Aopen once I've got the ability to create one there